Test/Utils/Testenv/Testenv.Impl.ps1

```diff
@@ -49,6 +49,7 @@ class Testenv {
 
         if ($InstallComponents) {
             foreach ($Testbed in $this.Testbeds) {
+                $Testbed.SetDataAdapterIpInfo()
```

Comment:
Why don't we do it in constructor?

Reply:
It's only neede if components are bing installed


There are only two places where this info is needed in clear test configuration and in initialize cnm plugin and extension. This is strictly used when components are being installed. I can do it both ways but it won't be used for anything.